doc/reference/usermode/memory_domain.rst

```diff
@@ -450,6 +444,8 @@ Some examples of partition attributes are:
     /* Denote partition is privileged read/write, unprivileged read-only */
     K_MEM_PARTITION_P_RW_U_RO
 
+In almost all cases ``K_MEM_PARTITION_P_RW_U_RW`` is the right choice.
```

Comment:
note: eventually (perhaps for LTS2) I would like to change things such that the application-facing APIs and data structure do not work with `k_mem_partition_attr_t`, and instead just have args for whether user mode can read, write, or execute, making it not possible for careless users to shoot themselves in the foot by changing supervisor mode access.